src/provisioner/config.py

```diff
@@ -58,9 +59,14 @@ def save(self, cortx_conf, cortx_solution_config):
         """ Save cortx-config into confstore """
 
         try:
-            cortx_conf.copy(cortx_solution_config)
+            cortx_solution_config_keys = []
+            for key in Conf.get_keys(cortx_solution_config):
+                if key.startswith('cortx'):
+                    cortx_solution_config_keys.append(key)
+            cortx_conf.copy(cortx_solution_config, cortx_solution_config_keys)
```

Comment:
Ensure that this code is not executed in upgrade case or we will end up overwriting all the keys. OR ensure that the target does not exist

Reply:
Yes this code is executed only in config apply case.